compiler/z/env/OMRCPU.hpp

```diff
@@ -122,6 +122,12 @@ class CPU : public OMR::CPU
     *     Determines whether the Transactional Memory (TM) facility is available on the current processor.
     */
    bool getSupportsTransactionalMemoryFacility();
+
+   /** \brief
+    *     Determines whether the Transactional Memory (TM) facility is available on the current processor.
+    *     Alias of getSupportsTransactionalMemoryFacility() as a platform agnostic query.
+    */
+   bool supportsTM();
```

Comment:
Not a fan of the duplication here, but until all the codegens have migrated to using the `CPU` class for CPU related queries we don't have a choice.